code/src/GridAPI/ConditionalFormat.ts

```diff
@@ -0,0 +1,24 @@
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+namespace GridAPI.ConditionalFormat {
+    /**
+     *
+     *
+     * @export
+     * @param {string} gridID
+     * @param {string} binding
+     * @param {number} rule
+     */
+    export function AddConditionalFormat(
+        gridID: string,
+        binding: string,
+        rule: Array<OSFramework.OSStructure.ConditionalFormat>
```

Comment:
SUPER COOL to have an API to immediately be able to do this!!!
We are missing is a RemoveConditionalFormat, it should be simple and straight forward.

Reply:
One thing that is not obvious is, what happens if with we add multiple rules to a binding? can we do that?
Imagine, I want the color to have an icon arrow that goes up and down depending on some condition and if the value is x I want to mark the line as red - these are 2 rules, However if I call the API twice the first rule is replaced (from what I saw in the code)...

Reply:
AddRule method expects an array of ConditionalFormat. So you can pass multiple rules to a binding.

![image](https://user-images.githubusercontent.com/3113318/117966800-3edc1900-b31c-11eb-8700-a5f85e379d1b.png)

In the example above, we have one rule for values between 20 and 50 and another for values below 100.